File: C++/1605.cpp
#include <iostream>
#include <stdio.h>
#include <cmath>
using namespace std;

int main() {
	int n;
	cin >> n;
	double ans = log10(2)*(n - 1);
	if (n % 2 == 0) {
		ans += log10(2);
	}
	cout << floor(ans) << endl;

	return 0;
}

File: C#/1605.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApplication2
{
    class Program
    {
        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());
            double ans = Math.Log10(2) * (n - 1);
            if (n % 2 == 0)
            {
                ans += Math.Log10(2);
            }
            Console.WriteLine(Math.Floor(ans));
        }
    }
}
